core/common/database/build.gradle.kts

```diff
@@ -5,4 +5,5 @@ plugins {
 dependencies {
     api(projects.core.common.coroutines)
     api(projects.core.common.model)
+    implementation(project(":core:common:conversion"))
```

Comment:
Adding this dependency here should not be necessary, let's figure out how to avoid it.